io.c

```diff
@@ -12055,7 +12054,7 @@ io_s_readlines(VALUE v)
  *
  *  - {Open Options}[rdoc-ref:IO@Open+Options].
  *  - {Encoding options}[rdoc-ref:encodings.rdoc@Encoding+Options].
- *  - {Line Options}[rdoc-ref:IO@Line+Options].
+ *  - {Line IO}[rdoc-ref:IO@Line+IO].
```

Comment:
Ditto here.

Reply:
Fixed.  This is _really_ careless.  The report showed both breaks.  Thanks for being a lert; we need more lerts.